### PR TITLE
Adds plain_log logger in wazuh-apid.py.

### PR DESCRIPTION
**Pull request by RamosFe**
|Related issue|
|---|
|https://github.com/wazuh/wazuh/issues/19199|

## Description

Fixes configuration of `plain` and `json` logs.

### CODE REVIEW

File: api/scripts/wazuh-apid.py [hunk 1]
```diff
@@ -213,9 +213,12 @@ def set_logging(log_path=f'{API_LOG_PATH}.log', foreground_mode=False, debug_mod
     log_path = f'{API_LOG_PATH}.log'
     foreground_mode = args.foreground
```
Comment (Selutario) [inline api/scripts/wazuh-apid.py:214]: This is already defined in each of the calls to `set_logging`.
```suggestion
```

**Review by Selutario — CHANGES_REQUESTED** [review]

File: api/scripts/wazuh-apid.py (hunk 1 above)
Comment (RamosFe) [reply api/scripts/wazuh-apid.py]: Deleted in commit d35012ab2665a1f75b9e47a67c7354eab8141573.

File: api/scripts/wazuh-apid.py [hunk 2]
```diff
@@ -213,9 +213,12 @@ def set_logging(log_path=f'{API_LOG_PATH}.log', foreground_mode=False, debug_mod
...
+    if plain_log:
+        set_logging(log_path=f'{API_LOG_PATH}.log', debug_mode=api_conf['logs']['level'],
+                    foreground_mode=args.foreground)
     if json_log:
-        log_path = f'{API_LOG_PATH}.json'
-        foreground_mode = args.foreground and not plain_log
+        set_logging(log_path=f'{API_LOG_PATH}.json', debug_mode=api_conf['logs']['level'],
+                    foreground_mode=args.foreground and not plain_log)
 
     set_logging(log_path=log_path, debug_mode=api_conf['logs']['level'], foreground_mode=foreground_mode)
```
[the remark this answers is not in the document]
Comment (RamosFe) [reply api/scripts/wazuh-apid.py]: Deleted in commit d35012ab2665a1f75b9e47a67c7354eab8141573.

File: api/scripts/wazuh-apid.py [hunk 3]
```diff
@@ -210,14 +210,13 @@ def set_logging(log_path=f'{API_LOG_PATH}.log', foreground_mode=False, debug_mod
...
+    if plain_log:
+        set_logging(log_path=f'{API_LOG_PATH}.log', debug_mode=api_conf['logs']['level'],
+                    foreground_mode=args.foreground)
     if json_log:
-        log_path = f'{API_LOG_PATH}.json'
-        foreground_mode = args.foreground and not plain_log
+        set_logging(log_path=f'{API_LOG_PATH}.json', debug_mode=api_conf['logs']['level'],
+                    foreground_mode=args.foreground and not plain_log)
 
-    set_logging(log_path=log_path, debug_mode=api_conf['logs']['level'], foreground_mode=foreground_mode)
```
Comment (Selutario) [inline api/scripts/wazuh-apid.py:220]: There should be only one blank line between logical sections of the same function ([PEP8](https://peps.python.org/pep-0008/#blank-lines)). Two blank lines can be used to, for example, separate different functions, but this is not the case.

```suggestion
```

**Review by Selutario — CHANGES_REQUESTED** [review]

**Review by Selutario — APPROVED** [review]

**Pull request merged**